storage/backend/boltoption_unix.go

```diff
@@ -30,5 +30,5 @@ import (
 // silently ignore this flag. Please update your kernel to prevent this.
 var boltOpenOptions = &bolt.Options{
 	MmapFlags:       syscall.MAP_POPULATE,
-	InitialMmapSize: InitialMmapSize,
+	InitialMmapSize: int(InitialMmapSize),
```

Comment:
No, that was not the reason for the test failure. I tested it with a 256MB mmap reservation and it still failed.

Likewise, go playground compiles with arch "amd64p32", which is why the ints are getting truncated. I tested on an amd64 arch and it gives the correct 64-bit value.